ui/components/component-library/component-library-components.scss

```diff
@@ -23,7 +23,6 @@
 // Molecules
 @import 'picker-network/picker-network';
 @import 'tag-url/tag-url';
-@import 'text-field/text-field';
 @import 'text-field-base/text-field-base';
 @import 'text-field-search/text-field-search';
 @import 'form-text-field/form-text-field';
```

Comment:
No longer needed now that the button styles have been removed see `ui/components/component-library/text-field/text-field.scss`